examples/language-sdk-instrumentation/golang-push/rideshare-alloy/README.md

```diff
@@ -45,7 +45,7 @@ pyroscope.write "backend" {
 # Pull latest images
 docker pull grafana/pyroscope:latest
 docker pull grafana/grafana:latest
-docker pull grafana/alloy:latest
+docker pull grafana/grafana/alloy-dev:latest
```

Comment:
We should be careful with use of dev images in examples, esp. `latest`. Users may not notice that and replicate it production.

```suggestion
docker pull grafana/alloy-dev:latest
```

Reply:
This was done as a temporary workaround since there's no other image available than the dev one, but I can pin it to a specific dev tag. 

The idea is to change it back to the previous state when the next release is available.